var/spack/repos/builtin/packages/rocdecode/package.py

```diff
@@ -33,6 +33,8 @@ class Rocdecode(CMakePackage):
         sticky=True,
     )
 
+    depends_on("libva", type="build", when=f"@{6.2:}")
```

Comment:
🤔 Does this actually produce what you want/expect?  Given Spack version range support, it *will* work for any `6.2.x` but won't for `6.3.x` when it is introduced.

```
$ python
>>> f"@{6.2:}"
'@6.2'
>>> "@6.2:"
'@6.2:'
```

Reply:
@srekolam, using an f-string as a constant string is superfluous. Just fix and de-obfuscate it with an open range:
```suggestion
    depends_on("libva", type="build", when="@6.2:")
```
See of course Tamara's review on what the result of the f-string was.